tools/bioformats2raw/bf2raw.xml

```diff
@@ -0,0 +1,127 @@
+    mkdir -p $output.extra_files_path;
+fi;
+
+#if str($io_options.input.ext)=='html':
+    #set dirlist=os.listdir($io_options.input.extra_files_path)
+    #assert len($dirlist) > 0
+    #set input_path=os.path.join(str($io_options.input.extra_files_path), str($dirlist[0]))
+#else:
+    #set input_path=$io_options.input
+#end if
+
+bioformats2raw $input_path $output.extra_files_path/$io_options.output_name
+#if $bf2raw_params.multiscales['options'] == 'auto':
+    --target-min-size $bf2raw_params.multiscales.min_xy_size
+#elif $bf2raw_params.multiscales['options'] == 'manual':
+    --resolutions $bf2raw_params.resolutions
+#end if
+--tile_height $bf2raw_params.tile_height
+--tile_width $bf2raw_params.tile_width
+--chunk_depth $bf2raw_params.chunk_depth
+--downsample-type $bf2raw_params.downsample_type
+--compression $bf2raw_params.compression
+--max_workers "\${GALAXY_SLOTS:-4}"
+$bf2raw_params.nesting
+$bf2raw_params.omefolder
+$bf2raw_params.droptop
+#if not str($bf2raw_params.dimension_order) == 'keep input order':
+    --dimension-order $bf2raw_params.dimension_order
+#end if
+###--overwrite
+--debug off &> /dev/null;
+
+    ]]></command>
+    <inputs>
+        <section name="io_options" title="Input-output paths">
+            <param name="input" type="data" format="atsf,scn,ali,labels,ffr,jpeg,ndpi,tf8,pty,fff,mnc,vms,mng,xml,pcx,img,pct,ims,bip,ome.xml,bin,flex,his,hdr,mov,psd,spi,ipl,dv,aiix,j2kr,pst,mod,ome.tif,tif,obf,dib,dic,im3,tga,pbm,c01,crw,mrcs,l2d,seq,mdb,cfg,htm,mvd2,arf,vsi,companion.ome,htd,aim,fts,ndpis,r3d.log,nef,res,jpx,ics,rec,nd2,cr2,dcm,mea,ome,lei,lms,j2k,oib,mtb,ima,ets,wlz,pict,sm3,sm2,nrrd,xdce,acff,al3d,zvi,1sc,xys,tiff,pgm,pcoraw,ppm,ipm,set,cxd,ipw,apl,fake,tnb,txt,xv,tf2,ps,log,zip,epsi,j2ki,dicom,top,msr,frm,hed,gif,dm2,dm3,dm4,zpo,wav,wat,2fl,sdt,liff,hx,pic,ome.btf,am,bmp,pnl,r3d_d3d,jpf,png,tfr,dti,nii.gz,cif,fdf,grey,df3,stk,fli,hdf,btf,stp,ch5,v,sld,ids,dv.log,jpk,mrw,r3d,xlog,ano,jpe,sxm,jpg,vws,raw,czi,spl,avi,ome.tf8,inf,spc,ome.tf2,spe,lsm,afm,lif,naf,afi,inr,lim,nd,tim,html,sif,env,tif,csv,map,nii,gel,ome.tiff,oif,par,amiramesh,pr3,fits,lut,jp2,oir,dat,aisf,zfr,zfp,xqd,eps,xqf,st,nhdr,i2i,thm,exp,svs,mrc" label="Input image located in Galaxy history"/>
+            <param name="output_name" type="text" value="data.zarr" label="Output zarr data"/>
+        </section>
+        <section name="bf2raw_params" title="Parameters fed to file conversion module">
+            <conditional name="multiscales" >
+                <param name="options" type="select" multiple="false" label="How should the number of resolutions be determined?" >
+                    <option value="auto" selected="true">auto</option>
+                    <option value="manual" selected="false">manual</option>
+                </param>
+                <when value="auto">
+                    <param name="min_xy_size" type="integer" min="1" optional="true" label="Min x-y dimension for the lowest resolution" value="256" />
```

Comment:
According to https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#coding-style:
```suggestion
                    <param name="min_xy_size" type="integer" min="1" value="256" optional="true" label="Min x-y dimension for the lowest resolution" />
```